### ISSUE TITLE
Triggers symbol highlight on selection change

### ISSUE THREAD

**Issue by AndreasArvidsson** [cursorless-dev/cursorless#2604]
In vscode we can trigger `editor.action.wordHighlight.trigger`

Only on pre and post actions?

**Comment by pokey** [comment]
Yeah interesting to think through when to do it. And we'd prob need a flag on `setSelections`, otherwise ide wouldn't know it's pre / post

**Comment by AndreasArvidsson** [comment]
Or a separate editor method?
